build-tools/config/webpack/webpack.conf.base.js

```diff
@@ -50,7 +50,7 @@ module.exports = (buildType = DEVELOPMENT, isDevelopment, options = {}) => {
     // single configuration properties go here
     // objects go into a separate file (e.g. webpack.partial.conf.entry.js)
     mode: buildType === PRODUCTION ? 'production' : 'development',
-    devtool: isDevelopment ? 'cheap-module-eval-source-map' : false,
+    devtool: isDevelopment ? 'eval-cheap-module-source-map' : false,
```

Comment:
I recently noticed that `isDevelopment` is never passed correctly from the `dev` webpack config, so this line always returns false. That's why you mentioned it works with the plugin even though you changed it.

If you also fix that issue, then you might want to change this just to an hardcoded `false` :)

Reply:
Checked, you're right. I console log the property and came as undefined.

I update the PR and now the property is coming as true if it is DEVELOPMENT mode.